src/hit/api/evaluator/rotations.cpp

```diff
@@ -13,7 +13,9 @@ using namespace std;
 
 namespace hit {
 
-    RotationSet::RotationSet(int num_slots) : num_slots_(num_slots) {
+    RotationSet::RotationSet(int num_slots, int post_btp_lvl) : num_slots_(num_slots) {
+        post_boostrapping_level = post_btp_lvl;
+        post_bootstrapping_scale = pow(2, default_scale_bits);
```

Comment:
Why aren't these in the initializer?

Reply:
Seems to be a weird C++ism. I tried to put them in the initializer list, but it seems you can't initialize superclass members that way.